tools/topology/topology1/sof/pipe-passthrough-playback-sched.m4

```diff
@@ -0,0 +1,57 @@
+# playback passthrough pipeline and dai
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_P --> B0 --> virtual-playback-passthrough
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+
+#
+# Components and Buffers
+#
+
+# Host "Playback" PCM
+# with 2 sink and 0 source periods
+W_PCM_PLAYBACK(PCM_ID, Playback, DAI_PERIODS, 0, SCHEDULE_CORE)
+
+
+# Playback Buffers
+W_BUFFER(0, COMP_BUFFER_SIZE(DAI_PERIODS,
+	COMP_SAMPLE_SIZE(PIPELINE_FORMAT), PIPELINE_CHANNELS, COMP_PERIOD_FRAMES(PCM_MAX_RATE, SCHEDULE_PERIOD)),
+	PLATFORM_HOST_MEM_CAP, SCHEDULE_CORE)
+
+# Virtual input widget
+VIRTUAL_WIDGET(VIRTUAL PLAYBACK PASSTHROUGH PIPELINE_ID, input, PIPELINE_ID)
+
+#
+# Pipeline Graph
+#
+#  host PCM_P --> B0 --> virtual-playback-passthrough
+
+P_GRAPH(pipe-passthrough-playback-sched, PIPELINE_ID,
+	LIST(`		',
+	`dapm(N_BUFFER(0), N_PCMP(PCM_ID))',
+	`dapm(VIRTUAL PLAYBACK PASSTHROUGH PIPELINE_ID, N_BUFFER(0))' ))
```

Comment:
Considering this adds the "VIRTUAL PLAYBACK" dapm, the filename should probably have "virtual" in the name. I.e. this "pipe-passthrough-playback-sched.m4" hints this is more generic pipeline than it actually is.

Reply:
added virtual in the file name.